pulp_ansible/app/downloaders.py

```diff
@@ -35,6 +44,7 @@ def __init__(self, *args, **kwargs):
 
 AUTH_TOKEN = None
 TOKEN_LOCK = asyncio.Lock()
+AUTOHUB_LIMIT_PER_HOST = 8
```

Comment:
I was rate limited with 8, then I changed to 2 and I could sync.
We need more tests to find the best number

Reply:
Yup, 8 was just a guess. Needs tuning.

Reply:
Updated to default to 2.